app.py

```diff
@@ -1,27 +1,121 @@
+from crypt import methods
+from curses.ascii import EM
+import email
 import os
 import json
 from flask import (
     Flask, flash, render_template,
-    redirect, request, session, url_for)
+    redirect, request, session, url_for, jsonify)
+import json
+from flask_login import current_user
```

Comment:
I took out Flask-login configuration after last week's "standup" call because you said you weren't completely set on Flask_Login.. I had already lost a few days of productivity because I was sick all week last week, so I was worried I was holding everyone up. 

I would remove the import, but your comment on the overall PR states that you do want to use flask-login again now right? I have to back-track a ton of commits to do that but I can do it